FTH/settings.py

```diff
@@ -126,10 +130,29 @@
 
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 STATIC_URL = '/static/'
-
+STATICFILES_DIRS = [
+        os.path.join(BASE_DIR,"/static/"),
```

Comment:
Wrong indentation, you could do it like this:
```suggestion
STATICFILES_DIRS = [
        os.path.join(BASE_DIR,"/static/"),
 ]
```